main.go

```diff
@@ -468,34 +468,35 @@ func runTest(buildTestParams models.XcodeBuildTestParamsModel, outputTool, xcpre
 	return rawOutput, exit, nil
 }
 
-func saveRawOutputToLogFile(rawXcodebuildOutput string, isRunSuccess bool) error {
+func saveRawOutputToLogFile(rawXcodebuildOutput string, isRunSuccess bool) (string, error) {
+	var logPth string
 	tmpDir, err := pathutil.NormalizedOSTempDirPath("xcodebuild-output")
 	if err != nil {
-		return fmt.Errorf("Failed to create temp dir, error: %s", err)
+		return logPth, fmt.Errorf("Failed to create temp dir, error: %s", err)
```

Comment:
in case of error we do not want to return any path

`return logPth, fmt.Errorf("Failed to create temp dir, error: %s", err)` -> `return "", fmt.Errorf("Failed to create temp dir, error: %s", err)`